src/controller/userRouter.js

```diff
@@ -35,6 +35,18 @@ userRouter.post("/login", validateUsername, validatePassword, async (req, res) =
     }
 });
 
+userRouter.put("/:userId", async (req, res) => {
```

Comment:
Need to add some middleware for authenticating that sender of request is the owner of the account, and for sanitizing request values